epersongroups.md

```diff
@@ -0,0 +1,307 @@
+  "name": "Administrator",
+  "handle": null,
+  "metadata": {
+  },
+  "permanent": true,
+  "type": "group",
+  "_links": {
+    "subgroups": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/subgroups"
+    },
+    "epersons": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/epersons"
+    },
+    "self": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc"
+    }
+  }
+}
+```
+
+## Create new EPerson Group
+
+**POST /api/eperson/groups**
+
+To create a new EPerson Group, perform a post with the JSON below to the epersons endpoint when logged in as admin.
+
+```json
+{
+  "name": "New Group 1",
+  "metadata": {
+      "dc.description": [
+        {
+          "value": "Test group",
+          "language": null,
+          "authority": "",
+          "confidence": -1
+        }
+      ]
+  }
+}
+```
+
+Status codes:
+* 201 Created - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions
+* 422 Unprocessable Entity - if the name was omitted or already exists, if permanent was set to true
+
+## Sub Groups in a single parent EPerson Group
+
+### Get Sub Groups in a single parent EPerson Group
+
+**GET /api/eperson/groups/<:uuid>/subgroups**
+
+Retrieves direct child groups of the given group.
+Requesting indirect child groups is not supported
+
+Sample:
+* GET /rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42/subgroups
+
+```json
+{
+    "_embedded": {
+      "groups": [
+        {
+          "id": "617cf46b-535c-42d5-9d22-327ce2eff6dc",
+          "uuid": "617cf46b-535c-42d5-9d22-327ce2eff6dc",
+          "name": "Administrator",
+          "handle": null,
+          "metadata": {},
+          "permanent": true,
+          "type": "group",
+          "_links": {
+            "subgroups": {
+              "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/subgroups"
+            },
+            "epersons": {
+              "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/epersons"
+            },
+            "self": {
+              "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc"
+            }
+          }
+        }
+      ]
+    },
+    "_links": {
+      "self": {
+        "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42/subgroups"
+      }
+    },
+    "page": {
+      "number": 0,
+      "size": 10,
+      "totalPages": 1,
+      "totalElements": 10
+    }
+}
+```
+
+### Add a Group to a parent Group
+
+**POST /api/eperson/groups/<:parentgroupuuid>/subgroups**
```

Comment:
Again, this could just be `/groups/<:parentgroupuuid>/groups` like the current implementation.  We don't seem to use "subgroups" anywhere...so you could do a bulk replace throughout this PR

Reply:
See https://github.com/DSpace/Rest7Contract/pull/41#discussion_r375177223